### PR TITLE
chore(blog) Rust banner & images

### PR DESCRIPTION
**Pull request by goodroot**
Whoops! Relates to: https://github.com/questdb/questdb.io/pull/1886

### CODE REVIEW

File: blog/2023-08-25-leveraging-rust-in-our-high-performance-java-database.mdx [hunk 1]
```diff
@@ -4,12 +4,18 @@ author: Adam Cimarosti
...
 slug: leveraging-rust-in-our-high-performance-java-database
 ---
 
-# Overview
+import Banner from "@theme/Banner"
+import Screenshot from "@theme/Screenshot"
+
+<Banner
+  alt="Rust and QuestDB! Rustaceans are welcome here."
+  src="/img/blog/2023-08-25/questdb_loves_java_and_rust_splash.webp"
```
Comment (argshook) [inline blog/2023-08-25-leveraging-rust-in-our-high-performance-java-database.mdx:17]:
```suggestion
  src="/img/blog/2023-08-25/banner.webp"
```

File: blog/2023-08-25-leveraging-rust-in-our-high-performance-java-database.mdx [hunk 2]
```diff
@@ -4,12 +4,18 @@ author: Adam Cimarosti
 author_url: https://github.com/amunra
 author_image_url: https://avatars.githubusercontent.com/u/1499096?s=96&v=4
 description: A guide to adding Rust to a Java codebase with JNI and the rust-maven-plugin.
-tags: [engineering, rust, java, jni, rust-maven-plugin, performance, zero-gc]
-image: /img/blog/2023-08-25/questdb_loves_java_and_rust_splash.jpg
+tags: [engineering, rust, java, rust-maven-plugin, performance, zero-gc]
+image: /img/blog/2023-08-25/questdb_loves_java_and_rust_splash.webp
```
Comment (argshook) [inline blog/2023-08-25-leveraging-rust-in-our-high-performance-java-database.mdx:8]:
```suggestion
image: /img/blog/2023-08-25/banner.webp
```

**Review by argshook — COMMENTED** [review]
if you were to rename `..._splash.webp` to just `banner.webp` and commit it, then during commit a thumbnail should be created automatically.

this way no need to keep also [this svg](https://github.com/questdb/questdb.io/blob/master/static/img/blog/2023-08-25/questdb_loves_java_and_rust_splash.svg)

**Comment by amunra** [comment]
The SVG is displayed at the bottom of the blog post.

You'd need to swap that out too if you want to get rid of it.

**Review by argshook — APPROVED** [review]

**Pull request merged**